python/sources/MQTT/main.py

```diff
@@ -71,34 +82,41 @@ def on_message(client, userdata, msg):
     payload = {
         "data": msg.payload.decode('utf-8'),
         "qos": str(msg.qos),
-        "timestamp": str(datetime.utcnow())
+        "timestamp": str(datetime.now(datetime.UTC))
```

Comment:
Is there no timestamp on MQTT message?

Reply:
Had a look, the timestamp is not something I can find a fix definition for. message timestamp wasn't in the protocol until v 5 either.